src/auto-evo/AutoEvoConfiguration.cs

```diff
@@ -0,0 +1,32 @@
+public class AutoEvoConfiguration : IRegistryType
+{
+    public int MUTATIONS_PER_SPECIES;
+    public bool ALLOW_NO_MUTATION;
+    public int MOVE_ATTEMPTS_PER_SPECIES;
+    public bool ALLOW_NO_MIGRATION;
```

Comment:
These should be named according to the normal C# field. Also converting these to properties with private set and the jsonproperty attribute would protect against someone trying to modify these during runtime.